pkg/cmd/release/create/create.go

```diff
@@ -442,6 +458,15 @@ func gitTagInfo(tagName string) (string, error) {
 	return string(b), err
 }
 
+func gitTagSha(tagName string) (string, error) {
+	cmd, err := git.GitCommand("rev-list", "-n", "1", tagName)
```

Comment:
To resolve a ref:
1. `rev-list` is an overkill for a single ref. `rev-parse` would be a better bet
2. It would be best to use fully-qualified refs, so `refs/tags/{tagName}`

Reply:
Was able to remove this call all together.